pyntc/devices/ios_device.py

```diff
@@ -222,13 +222,16 @@ def _wait_for_device_reboot(self, timeout=3600):
         while time.time() - start < timeout:
             try:
                 self.open()
-                self.show("show version")
```

Comment:
you would need `self.show("show version")` still right?

Reply:
If I'm not wrong, it is still being executed when getting the `uptime_string`. I think a need a different way of getting it though.

Reply:
This commit should make it work https://github.com/networktocode/pyntc/pull/268/commits/c31aebacdce62bf011e2c8df5e449cab53265920

Reply:
Thanks @balmasea , this looks like a good option.